.github/ISSUE_TEMPLATE/new-feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New feature request
```

Comment:
We already have a feature request template, right?

Reply:
This is surprising, I renamed the template and GitHub didn't erase the old one. To be clear, I don't write these by hand -- they're all generated by a GitHub tool. Surprised it doesn't handle renames...

Reply:
updated by hand